app/models/BasicPokemonDetail.js

```diff
@@ -9,6 +9,182 @@ let BasicPokemonDetailSchema = new mongoose.Schema({
     },
     name: {
         type: String
+    },
+    gender: {
+        /*male: {
```

Comment:
i would recommend make gender as a string, so that it will be like gender: "male" in the json, also for category, and move